client/state/a8c-for-agencies/types.ts

```diff
@@ -22,6 +22,15 @@ export interface Agency {
 			email: string;
 			name: string;
 			pressable_id: number;
+			usage: null | {
```

Comment:
Instead of expecting `null`, Maybe we can assume `undefined` here instead, so we only need to do something like:

```
usage?: {
				status: string;
				storage_gb: number;
				visits_count: number;
				sites_count: number;
				start_date: string;
				end_date: string;
				created_at: number;
			};
```